rllib/core/learner/torch/torch_learner.py

```diff
@@ -157,11 +158,13 @@ def add_module(
         )
 
         # we need to ddpify the module that was just added to the pool
-        self._module[module_id].to(self._device)
-        if self.distributed:
-            self._module.add_module(
-                module_id, TorchDDPRLModule(self._module[module_id]), override=True
-            )
+        module = self._module[module_id]
+        if isinstance(module, torch.nn.Module):
```

Comment:
use `if isinstance(module, TorchRLModule):` instead because we are still calling rlm.forward_train.

Reply:
done.